dev-python/pyside/pyside-9999.ebuild

```diff
@@ -135,8 +142,27 @@ src_configure() {
 
 	configuration() {
 		local mycmakeargs=(
+			-DPYTHON_EXECUTABLE="${PYTHON}"
 			"${mycmakeargs[@]}"
 		)
+
+		if python_is_python3; then
+			python_export PYTHON_CONFIG
+
+			# Extension tag unique to the current Python 3.x version (e.g.,
+			# ".cpython-34m" for CPython 3.4).
+			local EXTENSION_TAG="$("${PYTHON_CONFIG}" --extension-suffix)"
+			EXTENSION_TAG="${EXTENSION_TAG%.so}"
+
+			mycmakeargs+=(
+				# Find the previously installed "Shiboken2Config.*.cmake" file
+				# specific to this Python 3.x version.
+				-DPYTHON_EXTENSION_SUFFIX="${EXTENSION_TAG}"
```

Comment:
*Crikey.* Did I overengineer this? At least on my system, `PYTHON_EXTENSION_SUFFIX` appears to be incorrectly set when multiple Python 3.x interpreters are available due to PySide2's `CMakeLists.txt` finding the `PythonLibs` and `PythonInterp` packages in the wrong order. Oddly, Shiboken2's `CMakeLists.txt` finds these packages in the right order. Somebody made a smello typo.

But, yeah... I overengineered this. The *right* solution is to submit an upstream issue begging for a PySide2 fix. `</sigh>`

I'll revert this. Let's try again.